app/src/main/java/com/google/maps/android/ktx/demo/MainActivity.kt

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.maps.android.ktx.demo
+
+import android.graphics.Color
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.coroutineScope
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+import com.google.maps.android.clustering.ClusterManager
+import com.google.maps.android.collections.GroundOverlayManager
+import com.google.maps.android.collections.MarkerManager
+import com.google.maps.android.collections.PolygonManager
+import com.google.maps.android.collections.PolylineManager
+import com.google.maps.android.data.Renderer.ImagesCache
+import com.google.maps.android.data.geojson.GeoJsonLineStringStyle
+import com.google.maps.android.data.geojson.GeoJsonPolygonStyle
+import com.google.maps.android.ktx.MapsExperimentalFeature
+import com.google.maps.android.ktx.awaitMap
+import com.google.maps.android.ktx.demo.io.MyItemReader
+import com.google.maps.android.ktx.demo.model.MyItem
+import com.google.maps.android.ktx.utils.geojson.geoJsonLayer
+import com.google.maps.android.ktx.utils.kml.kmlLayer
+import org.json.JSONException
+
+/**
+ * A demo of multiple layers on the map.
+ *
+ * To add your Maps API key to this project:
+ *   1. Create a file app/gradle.properties
+ *   2. Add this line, where YOUR_API_KEY is your API key:
+ *        MAPS_API_KEY="YOUR_API_KEY"
+ */
+class MainActivity : AppCompatActivity() {
+    var mIsRestore: Boolean = false
```

Comment:
This doesn't need to be an instance property since it's only used in one place (in onCreate).

Reply:
Side note: I think we should also move away from using hungarian notation for naming variables since Android Studio can help visually distinguish between instance vs. global vs. local variables. A lot of old sample code still uses this notation but we it should be migrated away.

Reply:
Sounds good - refactored in https://github.com/googlemaps/android-maps-ktx/pull/29/commits/5e4a3253c8e258045a8060ce0bf9c6a9597850c3